lib/rubies/game.rb

```diff
@@ -155,6 +165,45 @@ def check_answer(current, input, target)
       end
     end
 
+    # deletes the target from the current data structure
+    def deep_delete(current, target)
+      return current unless current.is_a? Enumerable
+      current.
+          reject  {|value| value == target}.
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Space between { and | missing.<br>Space missing inside }.